### ISSUE TITLE
Patch wlancommID for mixed-region local multiplayer

### ISSUE THREAD

**Issue by vaguerant** [LumaTeam/Luma3DS#240]
A lot of games (e.g. Mario Kart 7) have region-locked local multiplayer. Trying to play cross-region is unsupported, and even if two different-region 3DS owners (e.g. USA + EUR) run the same game version (e.g. EUR + EUR), the EUR game will throw an error on the USA console when trying to enter local multiplayer.

I brought this up on IRC a while ago and got the following response:

```
Apr 15 01:00:35 <yellows8> patching the wlancommID should be enough for that.
```

Any hope for this?

**EDIT:** Actually, putting a hold on this. It looks like more changes might be required than just the above.

**Comment by marcotyschneider** [comment]
Hi is there any update on this? I'm having the similar issue. JPN o3ds with USA MK7 works fine, but EUR n3ds with USA MK7 throws an error ("A communication error has occurred") when trying to enter local multiplayer. Tried using luma region emulation but no change.

**Comment by matysc** [comment]
Hi is there any update on this? I'm having the similar issue. JPN o3ds with USA MK7 works fine, but EUR n3ds with USA MK7 throws an error ("A communication error has occurred") when trying to enter local multiplayer. Tried using luma region emulation but no change.
Update:
So I installed luma region switcher cia and it stops throwing the error :) However the two consoles are unable to detect the local multiplayer rooms hosted by the other console. Any idea?
